workflows/rnaseq/downstream/rnaseq.Rmd

```diff
@@ -154,17 +154,15 @@ Here is the sample table with metadata used for this analysis:
 knitr::kable(colData[, colnames(colData)[!colnames(colData) %in% exclude.for.printing]])
 ```
 
-```{r salmon}
+```{r salmon, eval=FALSE}
```

Comment:
if the default below is to use `salmon.files=txi` then this should be `eval=TRUE`, right?

Reply:
Not sure if I remembered correctly but set salmon.files=NULL in the example call instead.

Reply:
Yep, that would do it. So let's keep this PR using featureCounts, and once other PRs are in we'll switch over to using salmon.